src/server/scripts/Northrend/Ulduar/HallsOfStone/brann_bronzebeard.cpp

```diff
@@ -19,79 +19,79 @@
 
 enum NPCs
 {
-    NPC_DARK_RUNE_PROTECTOR         = 27983,
```

Comment:
Please restore the indentation for all enums to their original state in order to improve readability.

Reply:
https://github.com/azerothcore/azerothcore-wotlk/blob/master/src/server/scripts/Northrend/Ulduar/HallsOfStone/brann_bronzebeard.cpp 
check from here ,all enums states does original